src/Sarif.FunctionalTests/Sarif.FunctionalTests.csproj

```diff
@@ -16,6 +16,12 @@
     <AssemblyOriginatorKeyFile>..\TestKey.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., build.props))\build.props" />
+  <PropertyGroup>
+    <!-- Override the TargetFrameworkVersion specified in build.props. This project must target
+         .NET 4.6 because the Sarif.Validator project, upon which this project depends, targets
+         that version. -->
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
```

Comment:
why do we need to target v4.6? does this change impact SDK consumers? (I assume not, as the validation logic isn't incorporated into the SDK proper)